model_training/local_prediction/README.md

```diff
@@ -34,8 +34,7 @@ Tested on
 1. To obtain predictions on new images, simply place them in `.\test_images\` and run the script.
 1. Run LabelImg and open the `.\test_images\` directory. You can now adjust the bounding boxes and subsequently save the corresponding `.txt` files. The images can then be used as training data for the model.
 1. Some images may face an issue where a `ZeroDivisionError` occurs when saving using LabelImg. Be sure to check for this by saving after a single edit to avoid losing all your work.
-   ```bash
-   ...
+   ```Python
```

Comment:
For this README file, could you help to add an additional newline between all starting ``` and the line before it? Right now the formatting/whitespace doesn't look too good as the code block is sticking to the line before it.